packages/server-web/src/client/components/TodoItem.vue

```diff
@@ -7,7 +7,7 @@
 			  :removeTodo="removeTodo(todo)"
         :toggleAddTodoItem="toggleAddTodoItem"
 			/>
-      <span :class="`status status-${todo.status || 'open'}`" @click.prevent="toggleStatus(todo)"></span>
+      <span :class="`status status-${todo.status || 'open'}`" v-on:click="toggleStatus(todo)"></span>
```

Comment:
`v-on:` can be shortened to `@`, if I understood it correctly. Could be changed in a separate PR as well, as there are multiple occasions with `v-on:` already.

Reply:
yes, i changed it back to v-on here because i think it's clearer what's happening than the @. would prefer v-on generally, to be honest.

Reply:
Ok! Consistency is better